EIPS/eip-1.md

```diff
@@ -44,17 +44,21 @@ Each EIP must have a champion - someone who writes the EIP using the style and f
 
-Once the champion has asked the Ethereum community whether an idea has any chance of acceptance a draft EIP should be presented as a [pull request]. This gives the author a chance to continuously edit the draft EIP for proper formatting and quality. This also allows for further public comment and the author of the EIP to address concerns about the proposal.
+Anyone may submit a draft EIP as a pull request to the EIPs repository. Drafts should be placed in the 'drafts' directory and named in the format `eip-draft-*author*-*title*.md`, where *author* is the Github user or organisation name from which the pull request is sent, and *title* is a short hyphen-separated title for the EIP draft.
 
-If the EIP collaborators approve, the EIP editor will assign the EIP a number (generally the issue or PR number related to the EIP), label it as Standards Track, Informational, or Meta, give it status “Draft”, and add it to the git repository. The EIP editor will not unreasonably deny an EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum philosophy.
+Drafts must be formatted as described in this EIP; a template is available in [eip-X.md](https://github.com/ethereum/EIPs/blob/master/eip-X.md). All required headers must be present and filled in correctly. For a draft, the `eip` field must match the filename, without the 'eip-' prefix or '.md' suffix (eg, for `eip-draft-vitalik-foo.md`, the `eip` field should read `draft-vitalik-foo`). Drafts must contain a `discussions-to` header, specifying the URL to a venue where collaborators can discuss the draft. This may be a Github issue in the EIPs repository, or any other venue as the author sees fit.
 
-Standards Track EIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification]. The EIP should be reviewed and accepted before an implementation is begun, unless an implementation will aid people in studying the EIP. Standards Track EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final.
+An automated process will check all pull requests; if the request passes the format checks and contains only additions or edits to drafts owned by the submitter, it will be automatically merged. No editorial control is exercised over the content of an EIP draft.
```

Comment:
Spam attacks possible?

Reply:
I believe "automatically merged" in this context means that an editor will manually merge the document without any other factor manually preventing the merging from happening. As in an editor's personally opinion about the EIP will not make a difference in whether or not it gets merged; if it is technically correct it gets merged automatically by an editor without un-needed delay.

Reply:
Already possible on GitHub and happened before. One way to avoid this getting into the repository would be to build a time-delay into the automated process for merging drafts. In case there is a spam-attack or other obvious abuse, there will be some time to correct it.

Reply:
There is the "automated process" mentioned. I'd rather label PRs with a bot than merging them.

Reply:
> I believe "automatically merged" in this context means that an editor will manually merge the document without any other factor manually preventing the merging from happening. 

Actually, I was suggesting an entirely automated process. If someone wants to fill the repository with useless drafts, there's relatively little harm in doing so, and we can back it out (delete files) if need be afterwards. Any harm from a spam attack is minimal and easily undone.

Reply:
I'm having flashbacks to the last spam attack we faced across Ethereum repos back at the end of 2016 I believe. lol

Reply:
We had issues on this repo because each spam tx took up an EIP number - fortunately that wouldn't be an issue here any longer. And it's trivial to revert a repo back to a commit before the spam was added.